docs/source/photonics/examples/vary_width.py

```diff
@@ -30,8 +30,14 @@
         resolutions = dict(core={"resolution": 0.1, "distance": 1})
 
         with tempfile.TemporaryDirectory() as tmpdirname:
-            mesh_from_OrderedDict(polygons, resolutions, filename=tmpdirname + '/mesh.msh', default_resolution_max=.6)
-            mesh = Mesh.load(tmpdirname + '/mesh.msh')
+            mesh_from_OrderedDict(
+                polygons,
+                resolutions,
+                filename=f'{tmpdirname}/mesh.msh',
+                default_resolution_max=0.6,
+            )
+
```

Comment:
```suggestion
            mesh_from_OrderedDict(polygons, resolutions, filename=f'{tmpdirname}/mesh.msh', default_resolution_max=.6)
```
The usage of f-strings is nice! I don't see the point in adding the line breaks here.